itest/addrs_test.go

```diff
@@ -649,6 +649,16 @@ func runMultiSendTest(ctxt context.Context, t *harnessTest, alice,
 		return nil
 	}, defaultTimeout/2)
 	require.NoError(t.t, err)
+
+	// We start out with two assets at Alice, one normal and one grouped. We
+	// then send it to Bob and to ourselves, with change being created. So
+	// in each round we turn one of the assets into 3 pieces (two
+	// self-transfers via addresses and one change output).
+	if runIdx == 0 {
```

Comment:
super-nit: This could use the `runIdx` and `numRuns` values directly, as-is it would be incorrect for `runIdx > 1`.

But we don't call this func in that way for any existing tests, so no need to change right now.